lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -42,18 +42,47 @@ class _SceneRender {
   }
 }
 
+class SVGClipRegistry {
```

Comment:
Do we still need the SVG registry? According to MDN Safari implemented `clip-path: path(...)` in 13:

https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path

Reply:
Omg seriously? Ugh, I can't believe I spent all this effort writing this class and implementing disposal methods and stuff, haha. I'll remove this and simplify.